Core/src/org/sleuthkit/autopsy/casemodule/services/TagNameDefiniton.java

```diff
@@ -35,9 +37,12 @@
 
     private static final String TAGS_SETTINGS_NAME = "Tags"; //NON-NLS
     private static final String TAG_NAMES_SETTING_KEY = "TagNames"; //NON-NLS    
+    private static final String STANDARD_NOTABLE_TAG_DISPLAY_NAMES = "Evidence,Notable Item,"
+            + "CAT-1: Child Exploitation (Illegal),CAT-2: Child Exploitation (Non-Illegal/Age Difficult),CAT-3: Child Exploitive"; // NON-NLS
```

Comment:
I believe Notable Item is supposed to replace Evidence. Also, why are only three of the CAT tags included here, instead of all five?

Reply:
At the point of this pull request there is no Notable Item tag being made, the evidence tag is still being made, and for static String definitions the Category class has not been moved to the Core module so those names are not accessible and I haven't moved the static text for Follow Up or created it for Notable Item.